doc.js

```diff
@@ -365,21 +365,21 @@ define('doc', ['event'], function(event) {
 			},
 
 			'checked' : function(value) {
-				if(!this.attr('type').match('(checkbox|radio)')) {
-					throw Error("This element is not checkbox or radio type");
-				}
-
-				if(value === null || value === undefined) {
-					return this.first().checked;
+				if(value !== undefined && typeof value !== 'boolean') {
+					throw Error("Parameter type is not valid");
 				}
 
-				if(typeof value !== 'boolean') {
-					throw Error("Parameter type is not valid");
+				if(typeof value === 'boolean') {
+					this.each( function (el) {
```

Comment:
Poderia fazer o `return` já aqui, sem precisar da linha do `return` após essa chamada.

```suggestion
					return this.each(function (el) {
```

Reply:
:hammer_and_pick: